### DISCUSSION TITLE
Namespaces & lldpd

### DISCUSSION THREAD

**Discussion by NetAppGator** [lldpd/lldpd#525] [Q&A]
I am wondering if anyone has attempted to modify lldpd so that is namespace aware and can operate across namespaces? If so, tips would be appreciated.

**Comment by vincentbernat** [comment]
AFAIK, this has not been attempted. It may be difficult to discover namespaces (they are the ones mounted on `/run/netns`, but that's not mandatory), there is the question of whatever a namespace should be considered as a chassis or not. lldpd considers there can only be one local chassis. If the answer to this question is no, it shouldn't be that difficult once we know how to enumerate the namespaces.
